webdriver/result.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+class Result():
```

Comment:
Extend `object` and put it in webdriver/transport.py.

I also think you should consider renaming this to `Response` to match the spec language. A “result” seems more like a generic type.